python3-tryton.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "python3-tryton",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"tryton[calendar]==6.6.1\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/85/04/7e85b09322f9e7bb808f48e7bdab49e5672c72796ed7d7a120003d6bab76/GooCalendar-0.7.2-py3-none-any.whl",
+            "sha256": "071e8ee6977f8b2b4a9d8321947745a6d587b046cc4dc9c19ad99962067ec31b"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/69/ca/9e9fa2e8be0876a9bbf046a1be7ee33e61d4fdfbd1fd25c76c1bdfddf8c4/pycairo-1.23.0.tar.gz",
```

Comment:
this is in the gnome runtime.

Reply:
This was generated by flatpak-pip-generator